frontend/packages/ux-editor/src/components/Properties/DataModelBindings.module.css

```diff
@@ -8,6 +8,8 @@
 }
 
 .dataModelBindings {
+  padding-top: var(--fds-spacing-1);
+  padding-bottom: var(--fds-spacing-1);
```

Comment:
I suggest adding the spacing to the wrapper class instead, and including a bit more spacing, `var(--fds-spacing-2)`, so it is similar to the originally styling. This ensures consistent spacing with other similar list elements in the config.
 
Now:
![image](https://github.com/user-attachments/assets/7b5853f6-5f89-425b-a479-259cef8d1d05)

Originally:
![image](https://github.com/user-attachments/assets/40528dc6-01bf-4740-8aac-9e4181e09f65)